### PR TITLE
PXB-2930 2.4.27 release notes and release note tasks (2.4)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   docs/release-notes.mdnew file:   docs/release-notes/2.4/2.4.27.mdmodified:   mkdocs-base.yml

### CODE REVIEW

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

File: docs/release-notes/2.4/2.4.27.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+| Install instructions | [Install Percona XtraBackup 2.4](https://docs.percona.com/percona-xtrabackup/2.4/installation.html) |
+| Download this version | [Percona XtraBackup 2.4](https://www.percona.com/downloads/Percona-XtraBackup-2.4/LATEST/) |
+
+[Percona XtraBackup for MySQL Databases](https://www.percona.com/software/mysql-database/percona-xtrabackup) enables MySQL backups without blocking user queries. Percona XtraBackup is ideal for companies with large data sets and mission-critical applications that cannot tolerate long periods of downtime. Offered free as an open source solution, Percona XtraBackup drives down backup costs while providing unique features for MySQL backups.
+
+Percona XtraBackup 2.4 does not support making backups of databases created in MySQL 8.0, Percona Server for MySQL 8.0, or Percona XtraDB Cluster 8.0. Download [Percona XtraBackup 8.0](https://www.percona.com/downloads/Percona-XtraBackup-LATEST/#) to make backups for these versions.
+
+## Release highlights
+
+This release contains fixes for `wsrep_sync_wait` and an xbcloud fix.
```
Comment (altmannmarcelo) [inline docs/release-notes/2.4/2.4.27.md]:
```suggestion
This release contains fixes for backing up PXC and an xbcloud fix.
```

File: docs/release-notes/2.4/2.4.27.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Bugs fixed
+
+* [PXB-2809](https://jira.percona.com/browse/PXB-2809): `wsrep_sync_wait<>0` caused a `Lock wait timeout exceeded` error (Thanks to Frank Well for providing the initial patch).
```
Comment (altmannmarcelo) [inline docs/release-notes/2.4/2.4.27.md]:
```suggestion
* [PXB-2809](https://jira.percona.com/browse/PXB-2809): `wsrep_sync_wait<>0` caused a `Lock wait timeout exceeded` error when issue against PXC (Thanks to Frank Well for providing the initial patch).
```

File: docs/release-notes/2.4/2.4.27.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+* [PXB-2840](https://jira.percona.com/browse/PXB-2840): Xbcloud attempted to create a non-existent bucket.
```
Comment (altmannmarcelo) [inline docs/release-notes/2.4/2.4.27.md]:
```suggestion
* [PXB-2840](https://jira.percona.com/browse/PXB-2840): Xbcloud was missing to set location constraint when creating a bucket on non-default endpoint.
```

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: docs/release-notes/2.4/2.4.27.md (hunk 1 above)
Comment (patrickbirch) [reply docs/release-notes/2.4/2.4.27.md]: Updated.

File: docs/release-notes/2.4/2.4.27.md (hunk 2 above)
Comment (patrickbirch) [reply docs/release-notes/2.4/2.4.27.md]: Updated.

File: docs/release-notes/2.4/2.4.27.md (hunk 3 above)
Comment (patrickbirch) [reply docs/release-notes/2.4/2.4.27.md]: updated.

File: docs/release-notes/2.4/2.4.27.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## Release highlights
+
+This release contains fixes for backup up Percona XtraDB Cluster and an xbcloud fix.
+
+## Bugs fixed
+
+* [PXB-2809](https://jira.percona.com/browse/PXB-2809): `wsrep_sync_wait<>0` caused a `Lock wait timeout exceeded` error when issued against Percona XtraDB Cluster (Thanks to Frank Well for providing the initial patch).
+
+* [PXB-2840](https://jira.percona.com/browse/PXB-2840): Xbcloud did not have a required set location constraint when creating a bucket on a non-default endpoint.
```
Comment (altmannmarcelo) [inline docs/release-notes/2.4/2.4.27.md]: for to mention, this was also a contribution, please add

Thanks to Jason Zareski for the patch.

Comment (patrickbirch) [reply docs/release-notes/2.4/2.4.27.md]: Done.

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Pull request merged**